### DISCUSSION TITLE
How to use/map/analyze .gpx files rather than geojson/csv etc.

### DISCUSSION THREAD

**Discussion by bouton** [giswqs/geog-312#118] [Q&A] [answered]
Thanks for this fab course.  These notes work great to import and map geojson and csv files.  Is there an extension to read and map gpx tracks and waypoints (read_gpx?) or do I have to convert them all to geojson - or geoparquet
Thanks

**Comment by bouton** [comment, marked as answer]
Got it - con.sql("SELECT * FROM ST_Read('myfile.gpx', layer='tracks');")

**Comment by giswqs** [comment]
Great to see that you figured it out on your own. Thank you for sharing the solution.
